src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -733,8 +738,14 @@ export default class PhotoField extends React.Component {
           </div>
           }
           {fieldView === 'initial' && <div className="drop-target-container">
-            <Dropzone className="drop-target" onDrop={this.onChange} accept="image/jpeg, image/jpg, image/png, image/tiff, image/tif, image/bmp">
-              <img alt="placeholder" src="/img/photo-placeholder.png"/>
+            <Dropzone
+              className={`drop-target ${this.state.dragging && 'dragging'}`}
```

Comment:
`Dropzone` has an `activeClass` prop but it only takes an inline style object vs. a className.